notebooks/vertex_matching_engine/solutions/matching_engine_ann.ipynb

```diff
@@ -0,0 +1,646 @@
+{
```

Comment:
<p>Create a setup script for that in scripts (something like: <code>setup_for_matching_engine.sh</code>) and describe in a markdown cell that user may need to invoke it. This way we have all the setup for all the labs in a single place. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/230/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>